src/main/kotlin/no/nav/bidrag/grunnlag/service/OppdaterGrunnlagspakkeService.kt

```diff
@@ -104,10 +136,10 @@ class OppdaterGrunnlagspakkeService(
         fun oppdaterAinntekt(ainntektRequestListe: List<PersonIdOgPeriodeRequest>): OppdaterGrunnlagspakke {
```

Comment:
historiskeIdenterMap er ikke aktuelt for ainntekt og sigrun?

Reply:
De setter ikke gamle forekomster til inaktiv. Vet ikke helt hvorfor...